packages/in_app_purchase/in_app_purchase_android/lib/src/billing_client_wrappers/billing_client_wrapper.dart

```diff
@@ -574,6 +565,10 @@ enum BillingClientFeature {
   @JsonValue('priceChangeConfirmation')
   priceChangeConfirmation,
 
+  /// Play billing library support for querying and purchasing with ProductDetails.
+  @JsonValue('fff')
```

Comment:
Why the odd name?

Reply:
It matches the constant value used by the [Android API](https://developer.android.com/reference/com/android/billingclient/api/BillingClient.FeatureType#PRODUCT_DETAILS).
There is a comment at line 558 mentioning that the JSON values match these constant values. Should I clarify this further in the code?
As for why the Android team picked `fff`; I have no clue.

Reply:
I just missed that comment in the context of the review diffs; that's sufficient. Thanks!